src/main/scala/codacy/plugins/test/PluginsTests.scala

```diff
@@ -74,7 +75,7 @@ object PluginsTests extends ITest {
             true
           }
         case Failure(exception) =>
```

Comment:
Can we be constant on the naming of exceptions? Either as `e` or `exception`

Reply:
Sure :)

Reply:
Done 👍